packages/react-tracking/components/TrackingProvider.tsx

```diff
@@ -57,16 +69,24 @@ export function TrackingProvider({
         inheritedTracker instanceof Promise
           ? await inheritedTracker
           : inheritedTracker;
-
-      if (resolvedTracker instanceof Function) {
-        return resolvedTracker({
-          timestamp: new Date().getTime(),
-          path: path.join("."),
-          ...event,
```

Comment:
i think i would prefer to keep the `...event,` spread, because if properties are added to `event` in the future, i would like them to _automatically_ get added to this object, instead of having to remember to add them here.

ideally the mindset is "im going to call resolvedTracker(event) but i need to tweak some things on that event first"

the cumbersome way to achieve that would be

```ts
async function track(event) {
  event.timestamp = ...
  event.path = ...
  return resolvedTracker(event);
}
```

but the nicer syntax is to use spreading.